staging/operator-lifecycle-manager/deploy/chart/templates/0000_50_olm_07-olm-operator.deployment.yaml

```diff
@@ -17,6 +17,11 @@ spec:
       labels:
         app: olm-operator
     spec:
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65534
+        seccompProfile:
+          type: RuntimeDefault
```

Comment:
Making direct changes to the staging/* directories without attaching [metadata as a commit trailer](https://github.com/openshift/operator-framework-olm/commit/1fa303db75656d1b1123633e173fee22d87f3f27) will result in the verify prowjob to unfortunately fail.

We'd either need to introduce this changes to OLM, and cherrypick the upstream commit here to make the verify check happy, or inject these securityContext blocks as downstream-only changes. I think my preference would be to push through the latter right now.